src/components/ChartContainer.js

```diff
@@ -295,44 +275,73 @@ const ChartContainer = forwardRef(
     }));
 
     return (
-      <div
-        ref={container}
-        className={"orgchart-container " + containerClass}
-        onWheel={zoom ? zoomHandler : undefined}
-        onMouseUp={pan && panning ? panEndHandler : undefined}
+      <TransformWrapper
+        initialScale={1}
+        ref={transformComponentRef}
+        disabled={!(!!zoom)}
```

Comment:
After Jake had mentioned it I think this is slightly easier to digest:

`disabled={!Boolean(zoom)}`

But at the same time, not really lol. Definitely optional.